builtin/game/item_entity.lua

```diff
@@ -36,6 +36,9 @@ core.register_entity(":__builtin:item", {
 	itemstring = '',
 	physical_state = true,
 	age = 0,
+	-- The items are mergeable as long as this property is not false,
+	-- so we use nil to save a few bytes unless it will be set.
+	mergeable = nil,
```

Comment:
@BlockMen Actually, I'm checking there for explicitly not false, which has the problem sapier is pointing out:

    (nil ~= false) == true
    (true ~= false) == true
    (false ~= false) == false

    nil == false
    true == true
    false == false

So the simple check outlined by sapier will behave differently to the implementation.